src/configurator/yargs.js

```diff
@@ -6,8 +6,8 @@ import definitions from "./definitions.js"
 
 yargs
   .version(() => version)
+  .usage("Usage: phenomic <command> [options]")
   .help()
-  .showHelpOnFail()
```

Comment:
It's not working at all since we doesn't do any validation with `yargs.check()`. But this is actually an interesting idea. I could move the validation part to `yarts.check()` and let yargs do the jobs.